_rules/iframe-not-focusable-has-no-interactive-content-akn7bn.md

```diff
@@ -122,10 +122,42 @@ This `iframe` element contains a link that is not part of its [sequential focus
 <iframe tabindex="-1" srcdoc="<a href='/' tabindex='-1'>Home</a>"></iframe>
 ```
 
+#### Inapplicable Example 5
+
+This `iframe` element is [inert][] because of its own `inert` [attribute value][].
+
+```html
+<iframe tabindex="-1" inert srcdoc="<a href='/'>Home</a>"></iframe>
+```
+
+#### Inapplicable Example 6
+
+Once the "Privacy policy details" button is activated, the `iframe` element becomes [inert][] because of the showModal() method, which causes the `iframe` to be [blocked by a modal].
+
+```html
+<iframe tabindex="-1" srcdoc="<a href='/'>Home</a>"></iframe>
+<div>
+    <button id="ppDetails">Privacy policy details</button>
+</div>
+<dialog id="ppDialog" aria-labelledby="dialogLabel">
+    <h2 id="dialogLabel">Privacy Policy</h2>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+    <button id="cancel">Cancel</button>
+</dialog>
+<script>    
+			const dialog = document.getElementById("ppDialog");
+			window.addEventListener('load', function(event) {
+    			dialog.showModal();
+			});
+</script>
```

Comment:
I'm not entirely happy with this example. You can easily get this in an inaccessible state. Before onload the page is inaccessible, and after the modal is closed too. I also think that using "DOMContentLoaded" would be better, since most tools start testing when onload is triggered. This ensures the modal is already open when automated testing starts. I also added a title attribute to the iframe, and removed the duplicate ID. Lastly, I adjusted whitespace so it's what our formatter expects.

```suggestion
<iframe id="myFrame" title="Links" srcdoc="<a href='/'>Home</a>"></iframe>
<div>
	<button id="ppButton" onclick="openDialog()">Privacy policy details</button>
</div>
<dialog id="ppDialog" aria-labelledby="dialogLabel">
	<h2 id="dialogLabel">Privacy Policy</h2>
	<p>We store no data.</p>
	<button id="cancel" onclick="ppDialog.close()">Cancel</button>
</dialog>
<script>
	const openDialog = () => {
		ppDialog.showModal();
		myFrame.tabIndex = '-1'
	}
	ppDialog.addEventListener('close', () => myFrame.tabIndex = 0)
	window.addEventListener('DOMContentLoaded', openDialog);
</script>
```